setup_np4intel_dev_env.sh

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
+#
+# Copyright 2018-present Open Networking Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+if [[ $EUID -eq 0 ]]; then
+   echo "This script should not be run as root, run it as the user who owns the Stratum source directory"
+   exit 1
+fi
+
+PULL_DOCKER=NO
+MOUNT_SSH=NO
+BAZEL_CACHE=$HOME/.np4intel_cache
```

Comment:
Did you encounter errors when using the normal cache? Bazel should handle this.

Reply:
I'd have to go back and check, but I remember having issues when building the normal stratum based on debian and the np4intel platform based on ubuntu.  It ends up sharing the cache between the two different build environments.

Reply:
Just checked it and it does cause problems using the same cache for normal debian builds and np4_intel ubuntu builds. I can get around the issue by doing a "bazel clean -expunge" and then a rebuild but that defeats the purpose of a cache.